src/mime.cpp

```diff
@@ -1161,7 +1170,7 @@ void mime::merge_attributes(attributes_t& attributes) const
                 throw mime_error("Parsing attribute failure at `" + attr_name + "`.");
             }
         }
-        else
+        // else  //for fixing the while loop 
```

Comment:
What does it suppose to fix? Why not to put the increment into the for statement itself?

Reply:
I founded that the code woked unnormally on the mingw toolchains on the windows platform.but if changed to the vs toolchains,it would be normal.